docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  react:
+    container_name: frontend
+    build:
+      context: ./frontend
+    ports:
+      - "3000:3000"
+   
+      
+
+  node:
+    container_name: backend
+    build:
```

Comment:
```
    env_file:
      - .variables.env
```

Reply:
Hi @Ando22 thanks for noticing it 
even I have tried adding env variable in docker-compose file still backend was unable to get the env variable 
but let me try adding env_file instead of env-variable

Reply:
Okay @Ando22